.github/workflows/crowdin-upload.yml

```diff
@@ -0,0 +1,24 @@
+name: Crowdin Action
+
+on:
+  push:
+    branches: [main]
```

Comment:
Might make sense to add the `paths` values for YAML/JSON files like https://github.com/mastodon/mastodon/blob/f3fca78756c02fb19f75a2a712d80fa712d32229/.github/workflows/lint-yml.yml#L15 without the `!` if you re-submit this